dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1,8276 +1,8279 @@
-components:
```

Comment:
Think something weird happened here. 
Would check that the versions of the tools used in https://github.com/dockstore/dockstore/blob/1.10.2/dockstore-webservice/pom.xml#L1058 on your system match what circle is using

Reply:
@denis-yuen Are you referring to yq version?

Reply:
Yeah, looks like

Reply:
@denis-yuen I just merged from develop, wondering if it's alright now. How do I check if they are matching?

Reply:
think it is just `yq --version`